db/schema.rb

```diff
@@ -124,6 +124,7 @@
     t.datetime "updated_at", precision: 6, null: false
     t.jsonb "logs"
     t.bigint "current_judge_id"
+    t.boolean "is_error", default: false, null: false
```

Comment:
What's this for?

Reply:
Will let it go as a non-blocking fix this time but shouldn't be any changes here if no migration files present.